packages/components/src/Page/Page.tsx

```diff
@@ -14,6 +14,10 @@ import { Button, ButtonProps } from "../Button";
 import { Menu, SectionProps } from "../Menu";
 import { Emphasis } from "../Emphasis";
 
+export type ButtonActionProps = ButtonProps & {
+  ref?: React.RefObject<HTMLDivElement>;
+};
```

Comment:
Not sure if there's a better way to declare this type? It's a button, but with an optional ref prop. 🤷